### ISSUE TITLE
Batch size for COCO

### ISSUE THREAD

**Issue by shinya7y** [zehuichen123/DDOD#5]
For WIDER FACE, the paper describes the batch size.
"The model is trained with a batch size of 24 on 6 Titan V100s."

For COCO, is the total batch size 32?
`tools/dist_train.sh coco_cfg/ddod_r50_1x.py 8`
`samples_per_gpu=4,`

Can we achieve similar AP with a total batch size of 16 and `lr=0.01`?

**Comment by ruiningTang** [comment]
I use the pre-trained weights of torchvision and the total batch size is fixed as 8(4x2). The final mAP is 41.4, which is similar to 41.6 mentioned in this repo.

**Comment by zehuichen123** [comment]
The batch size and learning rate are followed with "scale-up" principle. I think a batch size of 16 along with lr=0.01 will yield similar results. The reason why I use bs=4, lr=0.02 is that it runs faster on my machine.

**Comment by shinya7y** [comment]
Thank you all!
I will try a total batch size of 16.

**Comment by shinya7y** [comment]
DDOD, R-50, 1x, batch size 16 (4x4), `lr=0.01`
`bbox_mAP_copypaste: 0.413 0.597 0.448 0.242 0.443 0.545`
They are similar but slightly worse, though the cause may not be the batch size and lr (e.g., randomness, instability due to fp16, implementation change of mixed-precision training (MMCV >= 1.3.2)).

I wonder how much the following update (fix?) affects accuracy.
```
- normal_init(self.atss_reg, std=0.01)
+ normal_init(self.atss_reg, mean=0.1, std=0.01)
```
https://github.com/zehuichen123/DDOD/commit/e1d7e11aca8e17ef173c737b880e96b75d982647#diff-d6a1e48f060020327909ffb9371be242cadf01128169158876f36d3ff6ce2ace

**Comment by zehuichen123** [comment]
The reason why I change this is that if we initialize atss_reg with \mu = 0 in FCOS, it will cause negative GIoU loss at the start of training.

**Comment by shinya7y** [comment]
Thank you for the clarification.
